src/Microsoft.IdentityModel.Tokens/Validation/ValidationFailureType.cs

```diff
@@ -39,6 +39,12 @@ private class IssuerValidationFailure : ValidationFailureType { internal IssuerV
         public static readonly ValidationFailureType AudienceValidationFailed = new AudienceValidationFailure("AudienceValidationFailed");
         private class AudienceValidationFailure : ValidationFailureType { internal AudienceValidationFailure(string name) : base(name) { } }
 
+        /// <summary>
+        /// Defines a type that represents that token type validation failed.
+        /// </summary>
+        public static readonly ValidationFailureType TokenTypeValidationFailed = new TokenTypeValidationFailure("TokenTypeValidationFailure");
```

Comment:
nit: All ValidationFailure names use the verb form `TokenTypeValidationFailed`.

Reply:
Updated along with LifetimeValidationFailed inconsistency. Thanks!